src/styles/home.css

```diff
@@ -0,0 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Federo&display=swap');
+:root {
+    --default-pink: #B24080;
+    --black: #000000;
+    --light-pink: #FFA6D5;
+    --white: #fff;
+    --gray: #495464;
+}
+body {
+    font-family: 'Poppins', sans-serif;
```

Comment:
For fonts also we can store them into variable inside root to access different fonts at different places